src/state/messages/convo/agent.ts

```diff
@@ -713,6 +741,8 @@ export class Convo {
       id: tempId,
       message,
     })
+    // remove on each send, it might go through now without user having to click
+    this.footerItems.delete(ConvoItemError.PendingFailed)
```

Comment:
Noticed while testing this that if a send fails, and you send another that succeeds, you'll have no pending messages but the retry error will still be present. In this case, which is now fixed, if you click retry, it simply removes itself and does nothing, since there are no pending messages left.